resources/nl-core/nl-core-MedicationDispense.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="nl-core-MedicationDispense" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicationDispense" />
+  <name value="NlcoreMedicationDispense" />
+  <title value="nl core MedicationDispense" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A dispense is the delivery of an amount of medicine to the patient, their administrator or their representative." />
+  <purpose value="A derived profile from [zib-MedicationDispense](http://nictiz.nl/fhir/StructureDefinition/zib-MedicationDispense) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="medicationdispense-v2.0.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/MedicationDispense-v2.0.2(2020EN)" />
+    <name value="zib MedicationDispense-v2.0.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="MedicationDispense" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicationDispense" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="MedicationDispense">
+      <path value="MedicationDispense" />
+      <alias value="nl-core-MedicationDispense" />
+    </element>
+    <element id="MedicationDispense.extension">
+      <path value="MedicationDispense.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationDispense.extension:medicationTreatment">

```

Comment:
Could you point to where this extension can be found?

Reply:
Because it is reused in a multiple profiles, it is placed in it's own branch.